main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
There is a typo in the actor's name 'Kianu Reaves'. It should be 'Keanu Reeves' to match the intended correction later in the code.